packages/react-native-editor/__device-tests__/helpers/utils.js

```diff
@@ -2,6 +2,7 @@
  * External dependencies
  */
 import childProcess from 'child_process';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Not sure how this is supposed to be handled on gutenberg, we can't have dev dependencies inside packages but there is this lint rule as well.
It seems there would be other packages where that would be a problem if they were modified today.

Reply:
Tests could be a special case where we import dev dependencies. What do you think about excluding this folder altogether from that rule?

Reply:
I'm not sure, feels like I'm missing some context on why we have that kind of rules in the first place. Let's just keep it for now